.github/workflows/operator-sdk.yaml

```diff
@@ -14,10 +14,10 @@ jobs:
           go-version: 1.16
       - run: make setup
       - run: make build
-      - run: sudo -E env PATH=${PATH}  go test -race -v $(go list ./... | grep -v vendor | grep -v e2e)
+      - run: make test
```

Comment:
should use sudo privileged mode.

but if you change to "sudo make test"  that use go 1.15 but not go 1.16. 

so please revert to  "sudo -E env PATH=${PATH} go test -race -v $(go list ./... | grep -v vendor | grep -v e2e)"

Reply:
- didn't get you on this point, why will `sudo make test` uses `go 1.15`?

Reply:
> 

see comment

Reply:
> 

the container of the github pipline has multi version of go. if you switch to root , the go version will be changed.

Reply:
- this makes sense, thanks a lot. I'll change it.